src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/ExcludeLinqExpressionFilterTests.cs

```diff
@@ -0,0 +1,168 @@
+
+            // Act
+            var results = sut.FilterMutants(mutants, null, new StrykerOptions()
+            {
+                ExcludedLinqExpressions = new List<LinqExpression>()
+                {
+                    exp
+                }
+            });
+
+            // Assert
+            results.ShouldBeEmpty();
+        }
+
+
+        [Theory]
+        [InlineData(LinqExpression.FirstOrDefault, LinqExpression.First)]
+        [InlineData(LinqExpression.First, LinqExpression.FirstOrDefault)]
+        [InlineData(LinqExpression.SingleOrDefault, LinqExpression.Single)]
+        [InlineData(LinqExpression.Single, LinqExpression.SingleOrDefault)]
+        [InlineData(LinqExpression.Last, LinqExpression.First)]
+        [InlineData(LinqExpression.All, LinqExpression.Any)]
+        [InlineData(LinqExpression.Any, LinqExpression.All)]
+        [InlineData(LinqExpression.Skip, LinqExpression.Take)]
+        [InlineData(LinqExpression.Take, LinqExpression.Skip)]
+        [InlineData(LinqExpression.SkipWhile, LinqExpression.TakeWhile)]
+        [InlineData(LinqExpression.TakeWhile, LinqExpression.SkipWhile)]
+        [InlineData(LinqExpression.Min, LinqExpression.Max)]
+        [InlineData(LinqExpression.Max, LinqExpression.Min)]
+        [InlineData(LinqExpression.Sum, LinqExpression.Max)]
+        [InlineData(LinqExpression.Average, LinqExpression.Min)]
+        [InlineData(LinqExpression.OrderBy, LinqExpression.OrderByDescending)]
+        [InlineData(LinqExpression.OrderByDescending, LinqExpression.OrderBy)]
+        [InlineData(LinqExpression.ThenBy, LinqExpression.ThenByDescending)]
+        [InlineData(LinqExpression.ThenByDescending, LinqExpression.ThenBy)]
+        [InlineData(LinqExpression.Reverse, LinqExpression.AsEnumerable)]
+        [InlineData(LinqExpression.AsEnumerable, LinqExpression.Reverse)]
+        [InlineData(LinqExpression.Union, LinqExpression.Intersect)]
+        [InlineData(LinqExpression.Intersect, LinqExpression.Union)]
+        [InlineData(LinqExpression.Concat, LinqExpression.Except)]
+        [InlineData(LinqExpression.Except, LinqExpression.Concat)]
+        public void ShouldNotRemoveLinqExpression(LinqExpression exp, LinqExpression excludedExp)
+        {
+            // Arrange
+            var target = new LinqMutator();
+
+            var expression = GenerateExpressions(exp.ToString());
+
+            var result = target.ApplyMutations(expression).ToList();
+
+            var mutants = result.Select(s => new Mutant() { ResultStatus = MutantStatus.Survived, Mutation = s });
+
+            var sut = new ExcludeLinqExpressionFilter();
+
+            // Act
+            var results = sut.FilterMutants(mutants, null, new StrykerOptions()
+            {
+                ExcludedLinqExpressions = new List<LinqExpression>()
+                {
+                    excludedExp
+                }
+            });
+
+            // Assert
+            results.ShouldNotBeEmpty();
+        }
+
+
```

Comment:
I'm missing a test with an empty ExcludedLinqExpressions. It should not filter anything.

Reply:
Added ShouldNotFilterMutationsWhenFilterIsEmpty test